### PR TITLE
Update core changes for 1.7.7

### PR DESCRIPTION
**Pull request by eternoendless**

### CODE REVIEW

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 1]
```diff
@@ -40,3 +147,12 @@ New hook                       | Function
...
 **actionGetAdminOrderButtons** | It is used to build a collection of buttons in the order thanks to an `ActionsBarButtonsCollection` which is automatically rendered
+
+## Other changes
+
+- CLDR `Specification\Number` [is now immutable](https://github.com/PrestaShop/PrestaShop/pull/15755)
+- Configuration forms have been split up [for security reasons](https://github.com/PrestaShop/PrestaShop/pull/17482)
+- New alias for webpack namespace app -> @app
+- The `clickable_row` option on Grids' `LinkRowAction` now uses `.clickable` as a selector, modules that rely on this feature should make sure that their templates use the appropriate class
+- CLDR data has been [updated to v36](https://github.com/PrestaShop/PrestaShop/pull/16347)
+- Smarty has been [updated to v3.1.31](https://github.com/PrestaShop/PrestaShop/pull/16429)
```
Comment (matks) [inline src/content/1.7/modules/core-updates/1.7.7.md:158]:
```suggestion
- Smarty has been [updated to v3.1.34](https://github.com/PrestaShop/PrestaShop/pull/16429)
```
3.1.31 to 3.1.34

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 2]
```diff
@@ -40,3 +147,12 @@ New hook                       | Function
 **displayAdminOrderSide**      | It is displayed in the side column (similar to displayAdminOrderSideBottom) but right under the customer view
 **displayAdminOrderMain**      | It is displayed in the side column (similar to displayAdminOrderMainBottom) but right under the details view
 **actionGetAdminOrderButtons** | It is used to build a collection of buttons in the order thanks to an `ActionsBarButtonsCollection` which is automatically rendered
+
+## Other changes
+
+- CLDR `Specification\Number` [is now immutable](https://github.com/PrestaShop/PrestaShop/pull/15755)
+- Configuration forms have been split up [for security reasons](https://github.com/PrestaShop/PrestaShop/pull/17482)
+- New alias for webpack namespace app -> @app
```
Comment (matks) [inline src/content/1.7/modules/core-updates/1.7.7.md:155]: Also we introduce modules namespace, dont know if worth mentioning

File: src/content/1.7/modules/core-updates/1.7.7.md [hunk 3]
```diff
@@ -7,6 +7,113 @@ aliases:
...
+symfony/swiftmailer-bundle | v2.6.7 | v3.2.6
+symfony/symfony | v3.4.26 | v3.4.37
+tecnickcom/tcpdf | v6.2.26 | 6.3.2
+tijsverkoyen/css-|-inline-styles | 2.2.1 | 2.2.2
+twig/twig | v1.38.4 | v1.42.4
+webmozart/assert | 1.4.0 | 1.6.0
+
+## New hooks
+
+The following hooks have been added:
```
Comment (matks) [inline src/content/1.7/modules/core-updates/1.7.7.md:93]: Do you want us to put here all new 1.7.7. hooks ? @tdavidsonas88 is working on a similar task

Comment (eternoendless) [reply src/content/1.7/modules/core-updates/1.7.7.md:93]: Yes, and also in the "all hooks" page

File: src/content/1.7/modules/core-updates/1.7.7.md (hunk 2 above)
Comment (eternoendless) [reply src/content/1.7/modules/core-updates/1.7.7.md:155]: I don't think it's worth it, that's why I left it out

**Comment by eternoendless** [comment]
Merging it as it is, it can be improved later on if necessary

**Pull request merged**